hybridse/include/vm/physical_op.h

```diff
@@ -1600,6 +1603,29 @@ class PhysicalDistinctNode : public PhysicalUnaryNode {
     virtual ~PhysicalDistinctNode() {}
 };
 
+class PhysicalLoadDataNode : public PhysicalOpNode {
+ public:
+    PhysicalLoadDataNode(const std::string &file, const std::string &db, const std::string &table,
+                         std::shared_ptr<node::OptionsMap> options)
+        : PhysicalOpNode(kPhysicalOpLoadData, false), file_(file), db_(db), table_(table), options_(std::move(options)) {}
+    ~PhysicalLoadDataNode() override = default;
+    void Print(std::ostream &output, const std::string &tab) const override;
+    base::Status InitSchema(PhysicalPlanContext *) override;
+    base::Status WithNewChildren(node::NodeManager *nm, const std::vector<PhysicalOpNode *> &children,
+                                 PhysicalOpNode **out) override;
+    static PhysicalLoadDataNode *CastFrom(PhysicalOpNode *node);
+
+    const std::string &File() const { return file_; }
+    const std::string &Db() const { return db_; }
+    const std::string &Table() const { return table_; }
+    const std::shared_ptr<node::OptionsMap> Options() const { return options_; }
+
+    std::string file_;
+    std::string db_;
+    std::string table_;
+    std::shared_ptr<node::OptionsMap> options_;
```

Comment:
We may return `node::OptionsMap` or provide another method to get `node::OptionsMap` because Java may not handle the pointer.

Reply:
OptionsMap is the map of ConstNode, I've checked `ConstNode` is worked, but forgot the map. I'll try to add it to swig.

Reply:
OptionsMap is too hard to be added in swig. And all %template class can't be added in namespace com.4paradigm.hybridse.xxx, they'll in com.4paradigm or com.4paradgim.std.
So I use the method `ConstNode* GetOption(string option)` to get the simple pointer `ConstNode*`.